taas/config.d/organizations.yml

```diff
@@ -4,30 +4,49 @@ sources:
         options:
             build_by_default: False
         beta-v1:
-            url: https://docs.google.com/spreadsheets/d/1oa7Wknz64dMQlV2XsW0ssjh6Gw5SnHVfGxDSw_5eKOQ/edit#gid=518938746
+            url: https://docs.google.com/spreadsheets/d/1GIA3FrRM1GZS3FoUDNpw35MhC_WLTdKBiiU2tUlWS84/edit#gid=0
             mapping:
-                id: HR info ID
+                id: ID
+                hrinfo_id:
```

Comment:
All our IDs refer to different things, so I agree with the having them all be separate fields.